modules/apps/27-interchain-accounts/types/keys.go

```diff
@@ -42,6 +42,12 @@ var (
 
 	// IsMiddlewareEnabledPrefix defines the key prefix used to store a flag for legacy API callback routing via ibc middleware
 	IsMiddlewareEnabledPrefix = "isMiddlewareEnabled"
+
+	// MiddlewareEnabled is the value used to signal that controller middleware is enabled
+	MiddlewareEnabled = []byte{0x01}
+
+	// MiddlewareDisabled is the value used to signal that controller midleware is disabled
+	MiddlewareDisabled = []byte{0x02}
```

Comment:
I'd be in favour of moving all the middleware enabled associated key/values to `controllertypes`. But I don't feel particularly strongly. Happy to leave in the top level `icatypes` as well.

Reply:
No preference! Whatever you think is best